src/ApiCompat/Microsoft.DotNet.ApiCompatibility/ApiComparer.cs

```diff
@@ -23,15 +24,38 @@ public class ApiComparer : IApiComparer
         public bool WarnOnMissingReferences { get; set; }
 
         /// <inheritdoc />
-        public Func<string?, string[]?, ComparingSettings>? GetComparingSettings { get; set; }
+        public ComparingSettings ComparingSettings
+        {
+            get => _comparingSettings ?? new ComparingSettings(includeInternalSymbols: IncludeInternalSymbols,
+                strictMode: StrictMode,
+                warnOnMissingReferences: WarnOnMissingReferences);
+            set => _comparingSettings = value;
+        }
+
+        public ApiComparer(bool includeInternalSymbols = false,
+            bool strictMode = false,
+            bool warnOnMissingReferences = false,
+            ComparingSettings? comparingSettings = null)
```

Comment:
Adding this constructor as it wasn't yet there and makes it less code to instantiate and configure an `ApiComparer`.